planetaryimage/image.py

```diff
@@ -274,7 +279,7 @@ def _parse_data(self, stream):
             data_stream.close()
 
     def _parse_band_sequential_data(self, stream):
-        data = numpy.fromfile(stream, self.dtype, self.size)
+        data = numpy.fromstring(stream.read(self.size*4), self.dtype)
```

Comment:
As, numpy.fromfile supports `open_file_type` and not the `gzip_open_fie_type`, numpy.fromstring used.